docs/source/Examples/lambci.rst

```diff
@@ -0,0 +1,312 @@
+	      # * task    directory of lambda function, /var/task in container
+	      # * runtime directory of the lambda runtime, /opt in container
+	      # * layers  directory of layer pool with each layer a sub directory
+	      # * env     environment file for lambci docker container
+	      directory : '~/lambci'
+	      # address of lambci server API
+	      address : 'localhost:9001'
+
+*******************************************************************************
+Install the function
+*******************************************************************************
+
+Install the ``echo`` example function using SRK with the following command:
+
+::
+
+	$ ./srk function create -s examples/echo
+
+
+*******************************************************************************
+Start the LambCI server
+*******************************************************************************
+
+SRK does not presently start the LambCI FaaS provider on its own, so you will
+need to start it manually. The following starts up a container
+running a LambCI server locally:
+
+::
+
+	LAMBCI_PATH=$HOME/lambci
+	LAMBCI_RUNTIME=python3.8
+	LAMBCI_HANDLER=lambda_function.lambda_handler
+	docker run --rm -d \
+	  --name srk-lambci \
+	  -v $LAMBCI_PATH/task:/var/task:ro,delegated \
+	  -v $LAMBCI_PATH/runtime:/opt:ro,delegated \
+	  --env-file $LAMBCI_PATH/env \
+	  -e DOCKER_LAMBDA_STAY_OPEN=1 \
+	  -p 9001:9001 \
+	  lambci/lambda:$LAMBCI_RUNTIME \
+	  $LAMBCI_HANDLER
+
+You may also want to create a script to streamline this process.
+See `Scripting LambCI Startup`_.
+
+*******************************************************************************
+Invoke the function
+*******************************************************************************
+
+Use SRK to invoke the ``echo`` function using the following command:
+
+::
+
+	./srk bench -b one-shot -a '{"hello" : "world"}'
+
+
+*******************************************************************************
+Shut down the LambCI server
+*******************************************************************************
+
+::
+
+	docker kill srk-lambci
+
+*******************************************************************************
+Using Custom Libraries
+*******************************************************************************
+
+SRK with LambCI allows you to add custom libraries to the runtime. The
+configuration procedure parallels that of AWS Lambda, which users
+`layers <https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html>`_.
+
+In this example we will create a runtime that includes Python's popular `Requests <https://requests.readthedocs.io/en/master/>`_.
+Use the commands below to install the layer:
+
+::
+
+	mkdir -p $HOME/lambci/layers/requests-layer/python
+	pip3 install requests -t $HOME/lambci/layers/requests-layer/python
+
+Now create a configuration file ``configs/lambci-local-requests.yaml`` as follows:
```

Comment:
This will break if #17 goes in as-is (removes the --config option). We can discuss that there so I guess it's fine to keep it here. We updated the simple 'hello world' section above to just use srk.yaml though FWIW.

Reply:
Updated